pkg/virt-controller/watch/preset.go

```diff
@@ -317,6 +362,7 @@ func applyPresets(vm *kubev1.VirtualMachine, presets []kubev1.VirtualMachinePres
 			annotateVM(vm, preset)
```

Comment:
https://github.com/kubernetes/kubernetes/blob/master/plugin/pkg/admission/podpreset/admission.go#L375

we should model annotation logic after Pod Presets.  The annotation is set for all presets, there's no ordering issues to consider.

Reply:
Agreed. I pushed a patch a couple hours ago that does this.

The issue was that previously, if no change to the VM occurred a preset was not considered to have been applied.